### ISSUE TITLE
Mistake in styles example

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments-migration-test#721]
(Original issue 1014 created by None on 2014-07-08T16:57:48.806313+00:00)

Hi,

when i play with formatting i go to page http://pygments.org/docs/styles/, where i found some examples. The forst one works as described, but the second one:

```
>>> from pygments.styles import get_style_by_name
>>> HtmlFormatter(style='colorful').style
<class 'pygments.styles.colorful.ColorfulStyle'>
```

Returns this:

```
>>> from pygments.styles import get_style_by_name
>>> HtmlFormatter(style='colorful').style
Traceback (most recent call last):
  File "<input>", line 1, in <module>
NameError: name 'HtmlFormatter' is not defined
```

It seems, that it has bad import clausule, which need to be:

```
>>> from pygments.formatters import HtmlFormatter
>>> HtmlFormatter(style='colorful').style
<class 'pygments.styles.colorful.ColorfulStyle'>
```
The simmilar issue is with the third example too.

regards

**Comment by Anteru** [comment]
(Original issue 1014 last updated on 2014-10-07T05:13:49.953345+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by Anteru** [comment]
(Original comment by tshatch on 2014-10-07T05:13:49.947481+00:00)

Fix style docs w/ HtmlFormatter example.

Fixes #1014

→ <<cset 933075ef39d7>>
